src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -152,7 +140,7 @@ public bool IsUnitQuaternion
         /// </summary>
         public bool IsRotation
```

Comment:
Well... I the code suggested me too much. You are correct. Rotation around non-unit quaternion doesn't make any sence because definion of the rotation uses unit quaternion. I will remove it in next commit ;)